CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
Are you planning to change this to Spring Boot application?
It will be easy to develop for other contributors, easy to run locally and moduler too.